README.md

```diff
@@ -55,11 +56,11 @@ Lowercase letters are typically used to begin variable names.
 - Additionally, unit testing encourages a more flexible and reusable code architecture. Developers are compelled to consider the different parts of their code in isolation and how they interact with one another when they design unit tests. As a result, the codebase is better planned and easier to maintain.[https://www.techtarget.com/searchsoftwarequality/answer/Is-unit-testing-an-important-aspect-of-software-development]
 
 # Community code
```

Comment:
Good work in identifying some potential issues. Have you started to work on any? I would like to see some code, and I would like to see you write about what you've done to solve one of these issues in your research & reflection journal.